Source/OxyPlot.Wpf/PlotView.cs

```diff
@@ -82,6 +82,7 @@ protected override void OnRender(DrawingContext drawingContext)
         /// <inheritdoc/>
         protected override void RenderOverride()
         {
+            RenderContext.TextMeasurementMethod = TextMeasurementMethod;
```

Comment:
Missing `this`

Reply:
Sorry, that one was a late addition, I'll fix it